Week 1/Weekly Practice Problems (Week 1)/2413: Smallest Even Multiple/anshu.py

```diff
@@ -0,0 +1,6 @@
+class Solution:
+    def smallestEvenMultiple(self, n: int) -> int:
+        ans = n
+        while not(ans % 2 == 0 and ans % n == 0):
```

Comment:
We actually dont need to loop.

If n is even, then n itself is the answer. If n is odd, then 2*n is the answer. :)